clusterloader2/pkg/measurement/common/bundle/test_metrics.go

```diff
@@ -218,26 +218,26 @@ func (t *testMetrics) Execute(config *measurement.Config) ([]measurement.Summary
 		appendResults(&summaries, errList, summary, executeError(t.etcdCPUProfile.String(), action, err))
 		summary, err = execute(t.etcdMemoryProfile, etcdGatherConfig)
 		appendResults(&summaries, errList, summary, executeError(t.etcdMemoryProfile.String(), action, err))
-		summary, err = execute(t.etcdMutexProfile, etcdGatherConfig)
```

Comment:
For Etcd I can see that mutex profiles are actually working, check https://gcsweb.k8s.io/gcs/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-scale-performance/1602710237461090304/artifacts/

Reply:
Yeah etcd actually enables both block and mutex: 
https://github.com/etcd-io/etcd/blob/8e3fc380dcaf44c37bb5dc19f1832981225061cc/pkg/debugutil/pprof.go#L28

I think we may have to introduce a new type called `BlockProfile`, enable that for all control plane components and then use `MutexProfile` only for etcd.

Reply:
Sure, sounds good.